solutions/USACO dec15-contaminated-milk.mdx

```diff
@@ -0,0 +1,117 @@
+---
+id: usaco-569
+source: USACO Bronze 2015 December
+title: Contaminated Milk
+author: Jesse Choe
+---
+
+[Official Analysis](http://www.usaco.org/current/data/sol_badmilk_bronze_dec15.html)
+
+<LanguageSection>
+<CPPSection>
+
+In this problem, we are asked to find how many people are sick based on what type of milk each person drank, when they drank it, and when a person got sick. To solve this problem, 
+we must find the types of milk that could (potentially) make a person sick. To do this, we must check if a person drank milk type X before they became sick. Therefore, we loop over
+every sick person and store the possible milk types in a 2D array of integers. Then, we loop over all the possible milk types for every sick person and check for commonalities. If
+there is consistency in the possible milk types (every sick person), then we add that to an array and consider it one of the possible types of milk that would make someone sick.
+After this, we check how many people drank one of the possible milk types (consistent with each sick person), and print the number of people who were sick.
+
+```cpp
+#include <bits/stdc++.h>
+
+using namespace std;
+using ll = long long;
+using vl = vector<ll>;
+using pl = pair<ll, ll>;
+using vpl = vector<pl>;
+using vc = vector<char>;
+using vs = vector<string>;
+using str = string;
+
+#define FastIO ios_base::sync_with_stdio(0); cin.tie(0)
```

Comment:
if possible, solutions should use as few macros as possible.

Reply:
Oops, I'm kinda used to writing with template LOL. Should I convert to without the macros?

Reply:
Preferably, yes.